test_vae_training/0_explore_data.py

```diff
@@ -0,0 +1,62 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:microbe] *
+#     language: python
+#     name: conda-env-microbe-py
+# ---
+
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+from cm_modules import paths
+
+# +
+# Load dataset
+# Each column is a bacterial taxon, and each row is a single sample.
+# Rows are named using the format "${PROJECT_ID}_${SAMPLE_ID}"
+# Each cell is a read count.
+# There are ~11,800 samples in the preliminary dataset.
+# sample x taxon
```

Comment:
Super nitpicky, but is the last line of the comment supposed to be there? It seems to say what you've already said.